pkg/tanka/load.go

```diff
@@ -84,6 +84,16 @@ func List(path string, opts Opts) ([]*v1alpha1.Environment, error) {
 	return loader.List(path, LoaderOpts{opts.JsonnetOpts, opts.Name})
 }
 
+// EvalJsonnet returns the raw evaluated Jsonnet
+func EvalJsonnet(path string, opts Opts) (interface{}, error) {
```

Comment:
This is functionally equivalent to `Eval(string, Opts) interface, error`. Let's get rid of this one and have a consistent external API (`Eval`, `Peek`, `Load`, `List`)